src/main/resources/db/migration/V001__Initial_Tables.sql

```diff
@@ -8,6 +8,9 @@ CREATE TABLE users
   facebook_id INT NOT NULL
 );
 
+ALTER TABLE users
+  ADD CONSTRAINT ck_gender CHECK (gender IN ('F', 'M', 'O'));
```

Comment:
Vi pratet om det, men legger inn diskusjonen her for andre som er interesserte.

Postgres og Slick støtter begge enums, men ikke swagger i Scalatra. Derfor må vi ty til manuelle constraints direkte i databasen.